common.blocks/menu/menu.bemhtml

```diff
@@ -13,10 +13,12 @@ block('menu')(
                             ctx.val.indexOf(val) > -1 :
                             ctx.val === val);
                 },
-                iterateItems = function(content) {
+                iterateItems = (function(content) {
                     var i = 0, itemOrGroup;
                     while(itemOrGroup = content[i++]) {
                         if(itemOrGroup.block === 'menu-item') {
+                            itemOrGroup.attrs = (itemOrGroup.attrs || {});
+                            itemOrGroup.attrs.id = itemOrGroup.attrs.id || this.generateId(); // Support a11y
```

Comment:
In any case, it's so weird to pass "attrs" field here – we have custom `id` field in other blocks (see [`button` docs][1]). Why should `menu-item` differ?

[1]: https://en.bem.info/libs/bem-components/v2.0.0/desktop/button/docs/#buttonid